build.gradle

```diff
@@ -40,7 +40,7 @@ def versions = [
         amClient             : '1.7.3',
         sendLetterClient     : '3.0.23',
         elasticSearch        : '7.17.15',
-        jacksonDatabind      : '2.15.2',
+        jacksonDatabind      : '2.16.0',
```

Comment:
there is not such version released yet - that's i had to put in supression file

Reply:
there is